lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html

```diff
@@ -42,6 +42,7 @@ <h3 class="adf-inherit-container-header">
                 data-automation-id="permission-info-button"
                 [adf-pop-over]="inheritedPermission"
                 [target]="target"
+                autofocusedElementSelector=".adf-sortable"
```

Comment:
I suggest using square brackets to align with other input properties in the template
```suggestion
                [autofocusedElementSelector]="'.adf-sortable'"
```

Reply:
done